### ISSUE TITLE
Accessibility - Dataverse - Fly Out Menus

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#102]
## Data Page - (Sample Page): VandeVusse-Mueller Cognitive Consent Form: https://data.qdr.syr.edu/file.xhtml?persistentId=doi:10.5064/F6YYA3O3/ID8HVJ&version=1.0 

Fly-Out Menu under blue Download File button - "Data File Citation" 

Resources: https://www.w3.org/WAI/tutorials/menus/flyout/
Mouse Users - Fly-Out menus should have a close delay of 1 second (for those with reduced dexterity), this one does not. 
Keyboard Users - fly-out menus shouldnt open with the tab key (and this one doesnt) but I am not sure about whether this menu is accessible for keyboard users. The article details 2 options for this. Its a bit above my understanding to know whether the fly out menu on our site is accessible via one of these two options

**Comment by qqmyers** [comment]
Not sure what the best option is. FWIW: This menu actually duplicates the 'Cite DataFile'/'Cite Data Project' links where the citations are shown. Since the source code for this is fairly plain (not a PrimeFaces component), I think the one second timeout and option 2: use parent as a toggle could be implemented here (right now the appearance is simple css - when you hover on the parent the child menu gets a display:'block' style and appears.)

**Comment by qqmyers** [comment]
Fixing as part of https://github.com/IQSS/dataverse/issues/8247, and deploying in advance to dev.
